api/src/libs/profile.ts

```diff
@@ -136,3 +136,24 @@ export const getClusters = async (profile: ProjectProfile): Promise<Cluster[]> =
     throw err;
   }
 };
+
+export const archiveProjectSet = async (profileId: number): Promise<void> => {
+  try {
+    // Step 1. Archive Contacts
+    ContactModel.findForProject(profileId).then(contacts => {
```

Comment:
Looks like you're switching styles from async await to traditional promise syntax. Might be making the code harder to read.

Reply:
Good catch, corrected!